src/main/java/com/scriptbasic/syntax/expression/BasicTagAnalyzer.java

```diff
@@ -42,7 +55,7 @@
     static {
         unaryOperatorMap.put("+", UnaryOperatorPlus.class);
```

Comment:
Perhaps this pull request could also assign a constant name to the unary and binary operators as well. Just a suggestion.